Data_Structures_Questions.md

```diff
@@ -33,8 +36,10 @@ Answer the following questions for each of the data structures you implemented a
 1. What is the runtime complexity of `ListNode.insert_after`?
 
 2. What is the runtime complexity of `ListNode.insert_before`?
+- A: 0(n)
 
```

Comment:
Ooh, are you sure about this? It's probably constant time. Remember, you do not have to traverse the entire linked-list to get to the end node!